_pages/community-service/index.html

```diff
@@ -9,13 +9,19 @@
     <h1>Community Service and Volunteering</h1>
     <p class="heading"><a href="/community-service/chinese">中文 (简体)</a> | <a href="/community-service/japanese">日本語</a> | <a href="/community-service/spanish">Español</a> | <a href="/community-service/portuguese">Português</a> | <a href="/community-service/german">Deutsche</a> | <a href="/community-service/french">Français</a> | <a href="/community-service/thai">ภาษาไทย</a> | <a href="/community-service/korean">한국어</a> | <a href="/community-service/italian">Italiano</a></p>
     <img src="/images/community-service/food-packing.jpg">
+    <br>
     <p>English Now! and our community service partner, the Washington Center for International Education (WCIE), organize service learning and volunteer programs year-round.</p>
     <p>Our goals are to connect the Washington, D.C. international community to the local community, and to help others. We strive to offer a variety of programs and are always open to suggestions! Check out our existing service learning programs <a href="https://washingtoncie.org/community-service-and-ssl/">here</a>. We hope you can join us! In addition, we encourage you to collaborate with English Now and WCIE to support these opportunities:</p>
-    <br> 
-    <p><strong>Online Food Drive</strong></p>
-    <p>Community Service focuses immediately on a basic need&mdash;food security. Many in our community may not have enough food, as the economic impact of the public health crisis hits.</p>
-    <p>Please consider participating in our <strong><q>Online Food Drive.</q></strong> Help by ordering online grocery deliveries to one of three great organizations. This food will go directly to people in need, in the community, through these organizations.</p>
-    <img src="/images/community-service/bethesdacares.png" class="logo">
+    <br>
+    <p><strong><a href="https://septemberhousemaj.org/">September House MAJ</a></strong> is a great organization and is a long-term English Now!/WCIE partner. September House MAJ provides support for everyday life for Japanese people living in the Mid-Atlantic region. The organization helps families overcome financial, social, and emotional hardship by providing them with consultations in Japanese, including assistance accessing appropriate public resources, and also including food assistance during this crisis.</p>
+    <p>September House MAJ has an Amazon Wish list that enables you to donate Japanese food to families in need. Click here:</p>
+    <p><a href="https://www.amazon.com/hz/wishlist/ls/1ZJA4TNX9M2KF?ref_=wl_share">https://www.amazon.com/hz/wishlist/ls/1ZJA4TNX9M2KF?ref_=wl_share</a></p>
```

Comment:
Nit: you can remove the query string from this URL. And also potentially the `www.`.